mods/cnc/rules/defaults.yaml

```diff
@@ -122,9 +122,12 @@
 		CloakDelay: 90
 		CloakSound: trans1.aud
 		UncloakSound: trans1.aud
-		RequiresCondition: cloak
+		RequiresCondition: cloak && !uncloak
 	ExternalConditions@CLOAK:
 		Conditions: cloak
```

Comment:
Bonus points if you could also change this to something like `cloak-crate-collected`